app/javascript/components/mentoring/testimonials-list/TrackDropdown.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback } from 'react'
+import { Icon, TrackIcon } from '../../common'
+import { useDropdown } from '../../dropdowns/useDropdown'
+import { Track } from '../TestimonialsList'
+
+const TrackFilter = ({
+  title,
+  iconUrl,
+  onChange,
+  checked,
+}: Track & { onChange: () => void; checked: boolean }) => {
+  return (
+    <React.Fragment>
+      <label className="c-radio-wrapper">
+        <input type="radio" onChange={onChange} checked={checked} />
+        <div className="row">
+          <div className="c-radio" />
+          <TrackIcon iconUrl={iconUrl} title={title} />
+          <div className="title">{title}</div>
+        </div>
+      </label>
+    </React.Fragment>
```

Comment:
I don't think you need the `React.Fragment`, as you already have one root element.

```suggestion
    <label className="c-radio-wrapper">
      <input type="radio" onChange={onChange} checked={checked} />
      <div className="row">
        <div className="c-radio" />
        <TrackIcon iconUrl={iconUrl} title={title} />
        <div className="title">{title}</div>
      </div>
    </label>
```

Reply:
Updated :+1: